src/stylus/main.styl

```diff
@@ -1,55 +1,143 @@
+//Imports from app theme
 @import 'nib'
 @import 'app/constants'
 @import 'allure-breakpoints'
 @import 'allure-mixins'
 @import 'app/mixins'
 
 
+//Custom styles for this plugin
+@import 'category-links'
+@import 'star-rating'
+@import 'share-buttons'
+@import 'product-grid-view.styl'
+
 .product_collection
```

Comment:
the CSS definitions in `main.styl` are a bit of a code smell. Every BEMy Block is its own file, and try and set up some folders so its not a huge list within `styles` folder so:

styles/...
styles/product-collection/...
styles/product/...

how does that sound?